.github/workflows/publish-documentation-next.yml

```diff
@@ -15,7 +19,11 @@ jobs:
       id-token: write
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - uses: grafana/writers-toolkit/publish-technical-documentation@publish-technical-documentation/v1
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          persist-credentials: false
+      - uses: grafana/writers-toolkit/publish-technical-documentation@9aed4f7c5b3caa1b58d72ee39232333407d4acbf # publish-technical-documentation/v1
+        # This internal action's version is left as a tag instead of a pinned hash because renovate
+        # does not support this type of tag/version format without custom configuration.
```

Comment:
```suggestion
```

This comment seems outdated, possibly it's outdated on the `main` branch too so feel free to ignore this suggetion.

Renovate supports slash-separated tags now (https://github.com/renovatebot/renovate/pull/35431) and the action is actually pinned to a hash.